src/styles/components/carousel/_base.scss

```diff
@@ -61,6 +40,29 @@ $scrollbar-max-width: 17px !default;
     margin-bottom: -1px;
   }
 
+  // We need to override slick sliders default scss
+  // so that we can have cool hover effects and not cut off
+  // tiles that overflow the slick container
+  .slick-list {
+    overflow: visible;
+  }
+
+  .slick-slide {
+    opacity: 0;
+    pointer-events: none;
+    transition: opacity 250ms ease;
+  }
+
+  // Align carousel items to left
+  .slick-track {
```

Comment:
I think this was needed for when we have a low number of slides in a carousel - like if you have 2 items in a carousel they're centered instead of aligned left.